public/app/features/plugins/admin/components/PluginDetailsRightPanel.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { PageInfoItem } from '@grafana/runtime/src/components/PluginPage';
+import { TextLink, Stack, Text } from '@grafana/ui';
+
+import { CatalogPlugin } from '../types';
+
+type Props = {
+  info: PageInfoItem[];
+  plugin: CatalogPlugin;
+};
+
+export function PluginDetailsRightPanel(props: Props): React.ReactElement | null {
+  const { info, plugin } = props;
+
+  return (
+    <div style={{ width: '20%' }}>
```

Comment:
Consider using the grid system (in combination with the place in my previous comment) instead of using percentages.

We should think of how this will look in a smaller screen